src/app/component/user/lower-nav-bar/lower-nav-bar.component.html

```diff
@@ -4,31 +4,59 @@
       <img [src]="logo" id="green-city-logo" alt="green-city-logo" />
     </a>
     <input class="menu-btn" type="checkbox" id="menu-btn" />
-    <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
+    <label class="menu-icon" for="menu-btn"
+      ><span class="navicon"></span
+    ></label>
     <ul class="menu">
       <li>
-        <a [routerLink]="['/welcome']" routerLinkActive="active-link" [routerLinkActiveOptions]="{exact:
-          true}" pageScroll href="#header">{{ "user.lower-nav-bar.home" | translate }}</a>
+        <a
```

Comment:
At least one attribute should be with a tag name. Example
<a  attr1
      attr2
      attr3>
  value
</a>

Reply:
Fixed